ElfUtils/ElfFile.cpp

```diff
@@ -200,10 +217,25 @@ std::string ElfFileImpl<ElfT>::GetFilePath() const {
   return file_path_;
 }
 
+template <typename ElfT>
+ErrorMessageOr<LineInfo> ElfUtils::ElfFileImpl<ElfT>::GetLineInfo(uint64_t address) {
+  auto line_info_or_error = symbolizer_.symbolizeCode(
+      *object_file_, {address, llvm::object::SectionedAddress::UndefSection});
+  if (!line_info_or_error) {
+    return ErrorMessage(absl::StrFormat(
+        "Unable to get line number info for \"%s\", address=0x%x: %s", object_file_->getFileName(),
+        address, llvm::toString(line_info_or_error.takeError())));
+  }
+
+  auto& line_info = line_info_or_error.get();
+  return LineInfo{line_info.FileName, line_info.Line};
```

Comment:
Out of curiosity: How does file name look like? Is it relative? Do I need to match with a prefix directory afterwards? Maybe this should be reflected in LineInfo name.

Reply:
You can see an example in the test. It seems like its the full path of the file at time of compilation